20250704 misc

peterthomassen opened the pull request.

peterthomassen committed:
Editorial clarification, fixes #12

peterthomassen committed:
Add overview of cases, and scenario descriptions

peterthomassen committed:
Fix registry column (incorrectly was boolean)

shuque reviewed (commented):
This looks fine. But we are really creating yet another registry, or proposing to add this column to the existing IANA DNSSEC algorithms registry (rfc8624-bis has already proposed to add their algo designations to that registry).

peterthomassen committed:
Editorial changes

peterthomassen committed:
Fix IANA considerations

peterthomassen commented:
> This looks fine. But we are really creating yet another registry, or proposing to add this column to the existing IANA DNSSEC algorithms registry (rfc8624-bis has already proposed to add their algo designations to that registry).

We're adding a column; the previous text was outdated. Corrected in c7b8dd2d5be67347f2b334497a0d18659ba07866.

I've also rebased on top of Duane's PR (which was merged just now), updated the date, added a change log, and clarified a random sentence: e7426ffc0b38e73cf2127e51b786d6308a8d59db

if you're OK with the IANA rewrite, I think this is ready for merge.
